### DISCUSSION TITLE
tomford15133

### DISCUSSION THREAD

**Discussion by tomford15133** [aclap-dev/video-downloadhelper#1126] [License & Purchase requests]
### Your question

Je l'ai supprimé par erreur - E-mail, veuillez le renvoyer, merci.

### Addon & CoApp version and details

Je l'ai supprimé par erreur - E-mail, veuillez le renvoyer, merci.

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

You should be posting this in the `License & Purchase requests` section of this forum.  Besides, there are instructions in the Wiki that you should follow.  There is a link to the Wiki at the top of every page of this forum.  Click it.  That will take you to the closest thing we have to a user manual.  You should get in the habit of referring to it regularly.. It shows a table of contents down the right side. Click the chapter with the title `FAQ`.  Read that.  You'll find a section that is relevant to your problem.  Follow those instructions.
